gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -918,6 +918,54 @@ public static ImageFilter notEquals(ImageField field, long value) {
   }
 
   /**
+   * Class for filtering disk lists.
+   */
+  class DiskFilter extends ListFilter {
+
+    private static final long serialVersionUID = 5856790665396877913L;
+
+    private DiskFilter(DiskField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static DiskFilter equals(DiskField field, String value) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method name and parameter number are suspiciously close to equals(Object)](https://www.codacy.com/app/mziccard/gcloud-java/file/2383968021/issues/source?bid=3140968&fileBranchId=3223208#l938)